future_challenges_for_RSEng.tex

```diff
@@ -360,18 +360,20 @@ \subsection{What are new tasks of RTPs?}
  FG-NOTE: RSEs are one type of specialization of RTPs ins science. Do we see any other?
 
 \subsection{Impact of Alternate Career Paths for Researchers on the Structure of Research Institutes}
-The RSE is probably one of the first professionalizations in the academic mid-level that is able
-to collectively voice their concerns. This enables the possibility to reshape universities.
-If more people are employed as mid-level, this makes a steadier pool of people
-available that can integrate new people that are regularly coming into departments and know the local facilities.
-On the other hand, an upcoming challenge how universities decide to design the contracts of people employed
-in RSE units. Will they be permanent, or will there also be an up-or-out mentality?
-
-As outlined in [cite RSE group paper] not all universities need to opt for strengthening
-the academic mid-level. Some might have chosen to set up external companies to outsource typical RSE tasks,
-thereby providing career paths outside of academia.
-
-FG-NOTE: By Nick. I've put it here, due to the relation with the RTP question.
+
+RSEs can either be embedded in research groups (belonging to the mid-level university staff),
+or be part of larger RSE units.
+The participants had the opinion that focusing on embedded RSEs would make onboarding new RSEs to existing projects easier.
+This, however, comes with additional funding challenges,
+as long-term funding is more difficult to acquire in lower levels of an organization.
+Third-party funding can mitigate this issue in the mid-term.
+However, the participants considered central RSE groups more likely in the future,
+due to the fact that there are already examples and activities from various universities in this direction
+(examples include the univerisities of Jena, Heidelberg, Göttingen).
+We expect funding in central groups to evolve from initially temporary positions to permanent positions,
+as the groups themselves prove themselves valuable.
+Embedded RSEs and central groups will need to network with each other,
```

Comment:
Do we want to cite our RSE group paper here? I think it'S likely, that it's at least on the de-RSE website, by the time review has started...

Reply:
Added a reminder as TODO.